common/src/test/scala/uk/ac/wellcome/models/MergedSierraRecordTest.scala

```diff
@@ -7,238 +7,276 @@ import uk.ac.wellcome.utils.JsonUtil
+      val originalRecord = MergedSierraRecord(id = "222")
 
-    val newRecord: Option[MergedSierraRecord] =
-      originalRecord.mergeBibRecord(record)
+      val newRecord: Option[MergedSierraRecord] =
+        originalRecord.mergeBibRecord(record)
 
-    newRecord.get.maybeBibData.get shouldEqual record
-  }
+      newRecord.get.maybeBibData.get shouldEqual record
+    }
 
-  it("should only merge bib records with matching ids") {
-    val record = sierraBibRecord(id = "333")
-    val originalRecord = MergedSierraRecord(id = "444")
+    it("should only merge bib records with matching ids") {
+      val record = sierraBibRecord(id = "333")
+      val originalRecord = MergedSierraRecord(id = "444")
 
-    val caught = intercept[RuntimeException] {
-      originalRecord.mergeBibRecord(record)
+      val caught = intercept[RuntimeException] {
+        originalRecord.mergeBibRecord(record)
+      }
+      caught.getMessage shouldEqual "Non-matching bib ids 333 != 444"
     }
-    caught.getMessage shouldEqual "Non-matching bib ids 333 != 444"
-  }
 
-  it("should be at version 1 when first created") {
-    val record = MergedSierraRecord(id = "555")
-    record.version shouldEqual 1
-  }
+    it("should always increment the version when mergeBibRecord is called") {
+      val record = sierraBibRecord(id = "666")
+      val originalRecord = MergedSierraRecord(id = "666", version = 10)
+      val newRecord = originalRecord.mergeBibRecord(record)
+      newRecord.get.version shouldEqual 11
+    }
 
-  it("should always increment the version when mergeBibRecord is called") {
-    val record = sierraBibRecord(id = "666")
-    val originalRecord = MergedSierraRecord(id = "666", version = 10)
-    val newRecord = originalRecord.mergeBibRecord(record)
-    newRecord.get.version shouldEqual 11
-  }
+    it("should return None when merging bib records with stale data") {
+      val record = sierraBibRecord(
+        id = "777",
+        title = "Olde McOlde Recorde",
+        modifiedDate = "2001-01-01T01:01:01Z"
+      )
 
-  it("should return None when merging bib records with stale data") {
-    val record = sierraBibRecord(
-      id = "777",
-      title = "Olde McOlde Recorde",
-      modifiedDate = "2001-01-01T01:01:01Z"
-    )
-
-    val mergedSierraRecord = MergedSierraRecord(
-      id = "777",
-      maybeBibData = Some(
-        sierraBibRecord(
-          id = "777",
-          title = "Shiny McNewRecordz 2.0",
-          modifiedDate = "2009-09-09T09:09:09Z"
+      val mergedSierraRecord = MergedSierraRecord(
+        id = "777",
+        maybeBibData = Some(
+          sierraBibRecord(
+            id = "777",
+            title = "Shiny McNewRecordz 2.0",
+            modifiedDate = "2009-09-09T09:09:09Z"
+          )
         )
       )
-    )
 
-    val result = mergedSierraRecord.mergeBibRecord(record)
-    result shouldBe None
-  }
+      val result = mergedSierraRecord.mergeBibRecord(record)
```

Comment:
Do we want to follow the same pattern as in: https://github.com/wellcometrust/platform/pull/1295? I.e. this returns an unchanged record?